### PR TITLE
cwebp: add page

### PR DESCRIPTION
**Pull request by stryaponoff**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known): 0.6.1**

### CODE REVIEW

**Review by mfrw — COMMENTED** [review]
@stryaponoff can you please sign the CLA :)

**Comment by stryaponoff** [comment]
> @stryaponoff can you please sign the CLA :)

I've already signed it, right after create of pull request

File: pages.ru/common/cwebp.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# cwebp
+
+> Конвертирует файл изображения в формат WebP.
+> Больше информации: <https://developers.google.com/speed/webp/docs/cwebp>.
+
+- Сжатие WebP со стандартными настройками (качество = 75) в сохранением в выходной ([o]utput) файл:
+
+`cwebp {{путь/к/изображению}} -o {{путь/к/результату.webp}}`
+
+- Сжатие WebP с наилучшим качеством ([q]uality) и наибольшим размером файла:
```
Comment (EmilySeville7cfg) [inline pages.ru/common/cwebp.md:10]:
```suggestion
- Сжать WebP с наилучшим качеством и наибольшим размером файла:
```

File: pages.ru/common/cwebp.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`cwebp {{путь/к/изображению}} -o {{путь/к/результату.webp}} -q {{100}}`
+
+- Сжатие WebP с наихудшим качеством и наименьшим размером файла:
```
Comment (EmilySeville7cfg) [inline pages.ru/common/cwebp.md:14]:
```suggestion
- Сжать WebP с наихудшим качеством и наименьшим размером файла:
```

File: pages.ru/common/cwebp.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`cwebp {{путь/к/изображению}} -o {{путь/к/результату.webp}} -q {{0}}`
+
+- Сжатие WebP с изменением размера изображения:
```
Comment (EmilySeville7cfg) [inline pages.ru/common/cwebp.md:18]:
```suggestion
- Сжать WebP, изменив размер изображения:
```

File: pages.ru/common/cwebp.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+`cwebp {{путь/к/изображению}} -o {{путь/к/результату.webp}} -resize {{width}} {{height}}`
+
+- Сжатие WebP с удалением информации о прозрачности:
```
Comment (EmilySeville7cfg) [inline pages.ru/common/cwebp.md:22]:
```suggestion
- Сжать WebP, удалив информацию о прозрачности:
```

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

File: pages.ru/common/cwebp.md (hunk 3 above)
Comment (stryaponoff) [reply pages.ru/common/cwebp.md:18]: I don't like this variant. It results into illiterate sentence. Russians can understand it but it's not Russian.

I propose to change the first part of the sentence and leave the second as is.

**Review by navarroaxel — APPROVED** [review]

File: pages/common/cwebp.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+- Compress a WebP with the worst [q]uality and smallest file size:
+
+`cwebp {{path/to/image_file}} -o {{path/to/output.webp}} -q {{0}}`
+
+- Compress a WebP and apply resize to image:
+
+`cwebp {{path/to/image_file}} -o {{path/to/output.webp}} -resize {{width}} {{height}}`
+
+- Compress a WebP and drop alpha channel information:
```
Comment (mfrw) [inline pages/common/cwebp.md:22]:
```suggestion
- Compress a WebP file and drop alpha channel information:
```

File: pages/common/cwebp.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+- Compress a WebP with the best [q]uality and largest file size:
+
+`cwebp {{path/to/image_file}} -o {{path/to/output.webp}} -q {{100}}`
+
+- Compress a WebP with the worst [q]uality and smallest file size:
+
+`cwebp {{path/to/image_file}} -o {{path/to/output.webp}} -q {{0}}`
+
+- Compress a WebP and apply resize to image:
```
Comment (mfrw) [inline pages/common/cwebp.md:18]:
```suggestion
- Compress a WebP file and apply resize to image:
```

File: pages/common/cwebp.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+- Compress a WebP with default settings (q = 75) to the [o]utput file:
+
+`cwebp {{path/to/image_file}} -o {{path/to/output.webp}}`
+
+- Compress a WebP with the best [q]uality and largest file size:
+
+`cwebp {{path/to/image_file}} -o {{path/to/output.webp}} -q {{100}}`
+
+- Compress a WebP with the worst [q]uality and smallest file size:
```
Comment (mfrw) [inline pages/common/cwebp.md:14]:
```suggestion
- Compress a WebP file with the worst [q]uality and smallest file size:
```

File: pages/common/cwebp.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
+# cwebp
+
+> Compress an image file to a WebP file.
+> More information: <https://developers.google.com/speed/webp/docs/cwebp>.
+
+- Compress a WebP with default settings (q = 75) to the [o]utput file:
```
Comment (mfrw) [inline pages/common/cwebp.md:6]:
```suggestion
- Compress a WebP file with default settings (q = 75) to the [o]utput file:
```

**Review by mfrw — APPROVED** [review]
@stryaponoff Thank you for your contribution.
LGTM :)

**Review by EmilySeville7cfg — APPROVED** [review]

**Comment by EmilySeville7cfg** [comment]
Merged, thanks! 😄

**Pull request merged**